news_items/021_Json_schema_comment/index.html

```diff
@@ -0,0 +1,41 @@
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+ .code-block {
+     width: 60%;
+     margin-left: auto;
+     margin-right: auto;
+     overflow: hidden;
+     margin-bottom: 1rem;
+     font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+     background: #f0f0f0;
+     padding: 1em;
+     padding-bottom: 0em;
+ }
+ .code-block div {
+     padding: 0.5rem;
+     display: block;
+     margin-bottom: -1em;
+     line-height: 125%;
+ }
+</style>
+
+<p>
+    PrairieLearn now add a new attribute <code class="user-output">comment</code>, in order to add comment for the code in JSON file. Here is an example to imply this attribute:
```

Comment:
This needs to be edited. I will do that before merging. Please leave this comment unresolved until I do it.

Reply:
I took a swing at it, but you can look.  (I did it before I realized this comment was for that file... :)

Reply:
Most appreciated! The less editing I need to do the better 😄